autoload/codeium.vim

```diff
@@ -15,7 +15,7 @@ let s:default_codeium_enabled = {
       \ '.': 0}
 
 function! codeium#Enabled() abort
-  if !get(g:, 'codeium_enabled', v:true) || !get(b:, 'codeium_enabled', v:true)
+  if !get(g:, 'codeium_enabled', v:true)
```

Comment:
Ah, my apologies. I didn't realize you'd have to modify this function. Maybe you can define a new function (`codeium#EnabledGlobally()`) that you can use instead? We need this function to not change.

Reply:
I don't have to modify it , I just thought that that the part I removed is a typo , or is it not ?

Reply:
oh my bad  I see now its buffer and global

Reply:
I restored it